.circleci/config.yml

```diff
@@ -142,7 +142,7 @@ jobs:
       - run:
           name: FFI header consistency check
           command: |
-            wget https://github.com/eqrion/cbindgen/releases/download/v0.9.1/cbindgen
+            wget https://github.com/eqrion/cbindgen/archive/v0.10.0.tar.gz -O cbindgen
```

Comment:
hm, looks like cbindgen is currently missing a binary release of v0.10.0.
I reached out to emilio to fix that.
In the meantime you could downgrade your local install (`cargo install cbindgen --ver "=0.9.1"`) and regenerate the headers with that.

We can land the cbindgen upgrade in a separate PR then.